examples/dreamcast/network/ntp/ntp.c

```diff
@@ -0,0 +1,118 @@
+/* KallistiOS ##version##
+ *
+ *   ntp.c
+ *   Copyright (C) 2023 Eric Fradella
+ *
+ *   This example demonstrates how to set the Dreamcast's
+ *   real-time clock to current UTC time via an NTP server.
+ *
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <netdb.h>
+
+#include <arch/arch.h>
+#include <arch/rtc.h>
+#include <kos/dbgio.h>
+#include <kos/net.h>
+
+#define NTP_PORT    "123"
+#define NTP_SERVER  "us.pool.ntp.org"
+#define NTP_DELTA   2208988800ULL
+
+#define ERR_EXIT() { thd_sleep(2000); exit(EXIT_FAILURE); }
+
+KOS_INIT_FLAGS(INIT_DEFAULT | INIT_NET);
+
+/* Structure for 48-byte NTP packet */
+typedef struct ntp_packet {
+    uint8_t leap_ver_mode;    /* First 2 bits are leap indicator, next
+                                 three bits NTP version, last 3 bits mode */
+    uint8_t stratum;
+    uint8_t poll_interval;
+    uint8_t precision;
+    uint32_t root_delay;
+    uint32_t root_dispersion;
+    uint32_t ref_id;
+    uint32_t ref_time_s;
+    uint32_t ref_time_f;
+    uint32_t orig_time_s;
+    uint32_t orig_time_f;
+    uint32_t rcv_time_s;
+    uint32_t rcv_time_f;
+    uint32_t trns_time_s;
+    uint32_t trns_time_f;
+} ntp_packet_t;
+
+int main(int argc, char **argv) {
+    ntp_packet_t packet;
+    int sockfd;
+    struct addrinfo *ai;
+    time_t ntp_time, dc_time;
+    /* Set the framebuffer as the output device for dbgio. */
```

Comment:
Oops... missed one....

```suggestion

    /* Set the framebuffer as the output device for dbgio. */
```